src/main/scala/com/etsy/sbt/Checkstyle.scala

```diff
@@ -10,56 +10,99 @@ import sbt._
 
 /**
   * An SBT plugin to run checkstyle over Java code
-  *  
+  *
   * @author Andrew Johnson <ajohnson@etsy.com>
+  * @author Alejandro Rivera <alejandro.rivera.lopez@gmail.com>
+  * @author Joseph Earl <joe@josephearl.co.uk>
   */
 object Checkstyle extends Plugin {
-  import com.etsy.sbt.Checkstyle.CheckstyleTasks._
+
+  object CheckstyleSeverityLevel extends Enumeration {
```

Comment:
This looks good.  I did see your comment about using Checkstyle's built in SeverityLevel enum instead.  I'm happy with either choice there.